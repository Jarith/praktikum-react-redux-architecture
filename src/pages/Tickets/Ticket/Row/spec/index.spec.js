import React from 'react';
import { Row } from '../';
import { mount } from 'enzyme';

describe('<Row />', () => {
    let wrapper;

    beforeEach(() => {
        const props = {
            origin: 'origin',
            destination: 'destination',
            startTime: 'startTime',
            endTime: 'endTime',
            duration: 100500,
            stops: 'stops',
            stopsCount: 100500,
        };

        wrapper = mount(<Row {...props} />);
    });

    it('Рендерится корректно', () => {
        expect(wrapper).toMatchSnapshot();
    });
});
