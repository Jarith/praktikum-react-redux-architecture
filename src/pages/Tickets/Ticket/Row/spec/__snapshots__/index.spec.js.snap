// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Row /> Рендерится корректно 1`] = `
<Row
  destination="destination"
  duration={100500}
  endTime="endTime"
  origin="origin"
  startTime="startTime"
  stops="stops"
  stopsCount={100500}
>
  <div
    className="ticket__row"
  >
    <div
      className="ticket__col"
    >
      <p
        className="ticket__label"
      >
        origin
         – 
        destination
      </p>
      <p
        className="ticket__value"
      >
        <time
          className="ticket__start-time"
        >
          startTime
        </time>
        –
        <time
          className="ticket__end-time"
        >
          endTime
        </time>
      </p>
    </div>
    <div
      className="ticket__col"
    >
      <p
        className="ticket__label"
      >
        В пути
      </p>
      <p
        className="ticket__value"
      >
        100500
      </p>
    </div>
    <div
      className="ticket__col"
    >
      <p
        className="ticket__label"
      >
        100500
      </p>
      <p
        className="ticket__value"
      >
        stops
      </p>
    </div>
  </div>
</Row>
`;
